apps/expo-go/metro.config.js

```diff
@@ -54,6 +54,22 @@ function withReactNativeLabPackages(config) {
   config.serializer.getPolyfills = () =>
     require(require.resolve('@react-native/js-polyfills', { paths: [reactNativeLabsPath] }))();
 
+  // Ensure the initialize code is being loaded from react-native-lab
+  const getMainModules = config.serializer.getModulesRunBeforeMainModule;
+  config.serializer.getModulesRunBeforeMainModule = () => {
+    // This module import should be loaded from `react-native-lab`
+    const initializeCore = 'react-native/Libraries/Core/InitializeCore.js';
+    const initializeCorePath = require.resolve(initializeCore, { paths: [reactNativeLabsPath] });
+    // Ensure the old initialize core path is replaced with the react-native-lab version
+    return getMainModules().map((filePath) => {
+      if (filePath.endsWith(initializeCore)) {
+        return initializeCorePath;
+      }
+
+      return filePath;
+    });
+  };
```

Comment:
cc @byCedric for this change

Reply:
Hey @Kudo! Cedric gave me this change

Reply:
cool, thank to both of you to work on this.